kobo/apps/stripe/tests/utils.py

```diff
@@ -30,14 +29,12 @@ def generate_plan_subscription(
         product_metadata = {**product_metadata, **metadata}
     product = baker.make(Product, active=True, metadata=product_metadata)
 
-    if not (price := Price.objects.filter(id=price_id).first()):
```

Comment:
This was making it difficult to test an org with multiple subscriptions and I couldn't figure out why it was written this way with a hardcoded id. Removing it doesn't seem to have broken any unit tests.

Reply:
Probably because sometimes `Baker` uses super long strings and gets truncated when inserting data in the DB. 
So, hardcoding the ID could be helpful when you need to retrieve the object.